src/modules/status.command.js

```diff
@@ -1,16 +1,13 @@
-const get = require('lodash/get');
 const command = require('./command');
 
-const status = async function (response, { i18n }) {
-    const { user } = response;
-    const curLvl = get(user, 'data.exp.lvl');
-    const exp = get(user, 'data.exp.value');
-    const expToNext = get(user, 'data.exp.nextLvl');
+const status = async function (response, context) {
+    const { i18n, getModuleData } = context;
+    const { lvl, value, nextLvl } = getModuleData('exp', context);
```

Comment:
ok ok ok... now it calling synchronously, but its still point find it unworked in nearby future then we rework... OH i dont know, ok? I WANT SLEEP AND WRITE VERY SHITTY